manifests/board-image/firmware-oerv-milkv-pioneer-LTS/2403.0.1.toml

```diff
@@ -0,0 +1,33 @@
+format = "v1"
+[[distfiles]]
+name = "sg2042_firmware_linuxboot.img.zip"
+size = 16982087
+urls = [ "https://fast-mirror.isrc.ac.cn/openeuler/openEuler-24.03-LTS-SP1/embedded_img/riscv64/SG2042/sg2042_firmware_linuxboot.img.zip",]
+restrict = [ "mirror",]
+
+[distfiles.checksums]
+sha256 = "1d1e47ce2cbbeda528c4338030f914c4905af89fc939783e3ff80fefa935c57d"
+sha512 = "0fdf299e372f8af01b988afcee3021045ab81d02e6bc3b3a90136f1f36ec3b4452f8fde25023f54bda65651d5cecfb3a9adfa216e8c9e001339a388f51941497"
+
+[metadata]
+desc = "Firmware image for Milk-V Pioneer and openEuler 24.03-LTS-SP1"
```

Comment:
This is LinuxBoot so make the package name and description more specific, by mentioning "linuxboot"/"LinuxBoot" instead of just "firmware". For example, we already say "uboot" instead of "firmware" elsewhere.